internal/engine/experiment/riseupvpn/riseupvpn.go

```diff
@@ -184,12 +186,20 @@ func (m Measurer) Run(ctx context.Context, sess model.ExperimentSession,
 	measurement.TestKeys = testkeys
 	urlgetter.RegisterExtensions(measurement)
 
-	caTarget := "https://black.riseup.net/ca.crt"
 	certPool := netx.NewDefaultCertPool()
 
-	multi := urlgetter.Multi{Begin: measurement.MeasurementStartTimeSaved, Getter: m.Getter, Session: sess}
-	inputs := []urlgetter.MultiInput{
-		{Target: caTarget, Config: urlgetter.Config{
+	// used multiple times below
```

Comment:
@cyBerta do you think this is okay? As far as I can see, it's safe to reuse a `multi`. Did you perhaps add multiple instantiation because the code was otherwise misbehaving? (If so, maybe I should document/fix it).

Reply:
Not sure anymore, but I'll test it.

Reply:
Thanks!

Reply:
works perfectly in miniooni and ooniprobe-cli

Reply:
Great!